S_RTOS_01IntroductionToFreertos/Task_2/main.c

```diff
@@ -88,22 +88,71 @@ static void prvSetupHardware( void );
 
 
 
+TaskHandle_t xLed_task_1_Handler = NULL;
+TaskHandle_t xLed_task_2_Handler = NULL;
+TaskHandle_t xLed_task_3_Handler = NULL;
 
 
+led_task_config task_1_cfg;
+led_task_config task_2_cfg;
+led_task_config task_3_cfg;
+
 
 /*
  * Application entry point:
  * Starts all the other tasks, then starts the scheduler. 
  */
 int main( void )
 {
+	/*global vars*/
+
+	task_1_cfg.pin_num = PIN1;
+	task_1_cfg.delay = 100;
```

Comment:
don't use the magic number

Reply:
https://github.com/arafaarafa/Sprints_Tasks/pull/10/files
all magic numbers deleted